app/models/passenger.rb

```diff
@@ -22,9 +18,15 @@ def self.deactivate_expired_doc_note
   def will_expire_within_a_week?
     expiration.present? && expiration <= 7.days.since && expiration >= Date.today
   end
-  def expired_within_3_days?
-    expiration.present? && expiration >= 4.days.ago && expiration <= 1.days.ago
+
+  def expired_within_grace_period? #will_expire_within_3_days?
```

Comment:
I see that you wrote a comment. I'm not sure what it means, but I'm guessing you're unsure of the method name.

Personally, I'd rather see `expired_within_grace_period?` than `will_expire_within_3_days?`. It helps me understand why you're checking if the thing (whatever it is) will expire within three days.